modules/contact/src/constraints/MechanicalContactConstraint.C

```diff
@@ -1727,7 +1738,7 @@ MechanicalContactConstraint::getTangentialPenalty(PenetrationInfo & pinfo)
   if (_normalize_penalty)
     penalty *= nodalArea(pinfo);
 
-  return penalty;
+  return penalty * MathUtils::pow(_penalty_multiplier, _augmented_lagrange_update_iteration);
```

Comment:
Can this be documented --if there is already some docs on this?

Reply:
Are you ok with me adding that in the follow on work for mortar AL?